README.md

```diff
@@ -1,33 +1,46 @@
-# july-challenge-tbd
+# Getting Started with Create React App
 
-A web app that acts as a directory to find employees based on their areas of expertise and team involvement.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-# Status:
+## Available Scripts
 
-_Planning and Designing_
+In the project directory, you can run:
 
-# Client-side:
+### `npm start`
 
-_Pending_
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-# Server-side:
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
 
-_Pending_
+### `npm test`
 
-# Technology
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-- Frontend Framework: React
```

Comment:
I feel like it could be helpful to keep this section in here (from line 19 to line 26)!